modules/va_facilities/README.v1.yml

```diff
@@ -445,9 +425,13 @@ paths:
         services. Only facilities of type "health" and "benefits" may be filtered by available
         services.
 
-        Results of this operation are paginated. JSON responses include pagination information
-        in the standard JSON API "links" and "meta" elements. GeoJSON responses include pagination
-        information in the "Link" header.
+        Results of this operation are paginated. Responses include pagination information
+        in the standard JSON API "links" and "meta" elements.
+
+        Also in the "meta" element is information about the drive time band that the requested location 
+        fell within for each the facility included in the response. The values of drive_time_band_min and 
+        drive_time_band_max are in minutes. For example, a facility returned with a matched drive_time_band_min 
```

Comment:
I don't think the implementation is currently set up for `drive_time_band_min`. Instead it is `drivetime_band_min` where `drivetime` is one word. The classes reflect that too with names like `DrivetimeBand` and the Postgres table name `drivetime_bands`. While that might not be correct, it is currently consistent. Should we make a final canonical call on whether it's `drive time` or `drivetime`?

Reply:
drivetime is good, would prefer to be consistent with existing code